### PR TITLE
Clarify the preflight port check output

### PR DESCRIPTION
**Pull request by sdodson**
Update informational header a bit.
Move deploymentconfigs to top of the list.

### CODE REVIEW

**Comment by sdodson** [comment]
@ncdc @ashcrow PTAL

Should we filter out inactive replication controllers?

File: playbooks/common/openshift-cluster/upgrades/files/pre-upgrade-check [hunk 1]
```diff
@@ -111,13 +111,18 @@ def print_validation_header():
...
+  Valid port names must:
+
+    * be less that 16 characters
     * have at least one letter
-    * only a-z0-9-
-    * do not start or end with -
-    * Dashes may not be next to eachother ('--')
+    * contain only a-z0-9-
+    * not start or end with -
+    * not contain dashes next to eachother ('--')
```
Comment (ncdc) [inline playbooks/common/openshift-cluster/upgrades/files/pre-upgrade-check]: need a space between "each" and "other"

**Comment by ncdc** [comment]
@sdodson by inactive RCs, do you mean those with replicas=0?

**Comment by sdodson** [comment]
Correct. I'm trying to look at the best, and ideally quickest, way to get
someone to a safe place to upgrade.

**Comment by ncdc** [comment]
@sdodson I think it's better to correct everything. If they upgrade, and then rollback a deployment to a previous RC that has an invalid port name, that would fail, wouldn't it? cc @ironcladlou

**Comment by sdodson** [comment]
@brenton this is ready

**Pull request merged**